src/components/PostList/PostList.jsx

```diff
@@ -1 +1,9 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} />
```

Comment:
Each `PostInfo` component in the list should have a unique `key` prop. Consider using a unique identifier from the `post` object, such as `post.id`, to set the `key` prop.